receiver/webhookeventreceiver/req_to_log.go

```diff
@@ -60,3 +67,21 @@ func appendMetadata(resourceLog plog.ResourceLogs, query url.Values) {
 		}
 	}
 }
+
+// append headers as attributes
+func appendHeaders(config *Config, scopeLog plog.ScopeLogs, headers http.Header) {
+	for k := range headers {
+		// Skip the required header used for authentication
+		if k == textproto.CanonicalMIMEHeaderKey(config.RequiredHeader.Key) {
+			continue
+		}
+		scopeLog.Scope().Attributes().PutStr(headerAttributeKey(k), strings.Join(headers.Values(k), ";"))
```

Comment:
You can put a slice there, no need to transform the headers into a string as a `;` separator will not work if any header value contains a `;`.

Reply:
ah I'm also missing a test case for multiple header values, will add one